### PR TITLE
Handle schedule overlap & document limitation

### PR DESCRIPTION
**Pull request by cdlliuy**
#157476214  #157801476

### CODE REVIEW

File: docs/Policy_definition.rst [hunk 1]
```diff
@@ -91,8 +91,32 @@ App-AutoScaler Policy Definition
 | initial_min_instance_count           | int                        | false   | the initial minimal number of instance count for this schedule             |
 +--------------------------------------+----------------------------+---------+----------------------------------------------------------------------------+
 
+**Note**
```
Comment (boyang9527) [inline docs/Policy_definition.rst]: change this section to "constraints"

File: docs/Policy_definition.rst [hunk 2]
```diff
@@ -91,8 +91,32 @@ App-AutoScaler Policy Definition
...
 
-**Reference**
+You need to aware the following facts when define multiple schedules in policy:
```
Comment (boyang9527) [inline docs/Policy_definition.rst]: remove this sentence, just list all the constraints.

File: docs/Policy_definition.rst [hunk 3]
```diff
@@ -91,8 +91,32 @@ App-AutoScaler Policy Definition
...
+
+* If one schedule overlaps another, the one which starts first will be guaranteed, while the later one is completely ignored. For example: 
+
++-----+---------------------------------------------------------------------------------------+
```
Comment (boyang9527) [inline docs/Policy_definition.rst]: not to use index, just use schedule 1, schedule 2

File: docs/Policy_definition.rst [hunk 4]
```diff
@@ -91,8 +91,32 @@ App-AutoScaler Policy Definition
...
+|Index|Schedule                                                                               |
++-----+---------------------------------------------------------------------------------------+
+|1    |8:00AM ~ 8:00PM every Tuesday , with instance_min_count=1, instance_max_count=10       |
```
Comment (boyang9527) [inline docs/Policy_definition.rst]: give an example of three schedules, 
```
schedule 1:      |-----------| 
schedule 2:             |----------------------|
schedule 3:                               |-------|
```
In this scenario,  schedule 1, 3 will be applied, schedule 2 will be ignored.

File: docs/Policy_definition.rst [hunk 5]
```diff
@@ -91,8 +91,32 @@ App-AutoScaler Policy Definition
...
+|2    |10:00AM ~ 10:00PM on 2019-01-01, with instance_min_count=5, instance_max_count=50      |
++-----+---------------------------------------------------------------------------------------+
+
+With above definition, on the day of 2019-01-01 (which is Tuesday), schedule #1 will be executed as it occurs first, and schedule #2 will be discarded. 
 
-`Sample policy <https://github.com/cloudfoundry-incubator/app-autoscaler/blob/develop/src/integration/fakePolicyWithSchedule.json>`_
+* If a schedule's start time is earlier than the policy creation/update time, the schedule will not be executed. For example: 
+
++-----+---------------------------------------------------------------------------------------+
+|Index|Schedule                                                                               |
```
Comment (boyang9527) [inline docs/Policy_definition.rst]: again remove index.

File: docs/Policy_definition.rst [hunk 6]
```diff
@@ -91,8 +91,22 @@ App-AutoScaler Policy Definition
...
+    - Schedule #3:  --------------------------sssssssss------------     
+
+    With above definition, schedule #1 and #3 will be applied, while scheudle #2 is ignored.
 
-`Sample policy <https://github.com/cloudfoundry-incubator/app-autoscaler/blob/develop/src/integration/fakePolicyWithSchedule.json>`_
+* If a schedule's start time is earlier than the policy creation/update time, the schedule will not be executed. For example: 
+
+    - Schedule #1:  09:00 - 13:00 , Everyday
+   
+    If above schedule is created at 10:00AM someday, it won't take effect on the today it creates, but will be certainly triggered on the next day.
```
Comment (boyang9527) [inline docs/Policy_definition.rst]: on the toady --> when it creates

**Comment by boyang9527** [comment]
LGTM

**Comment by paltanmoy** [comment]
LGTM

**Pull request merged**